back-end/src/main/kotlin/Gneiss/PacketCompiler/DatabaseAccess/PacketDao.kt

```diff
@@ -31,4 +31,13 @@ class PacketDao(jsonSerializer: IJsonSerializer) : IPacketDao {
         }
         return ret
     }
+
+    override fun delete(key: String, field: String, packet: Packet): Long {
+        val jedis = pool.getResource()
+        var ret: Long
+        jedis.use {
+            ret = jedis.hdel(key, field, jsonSerializer.serializePacket(packet))
```

Comment:
Since hdel returns 1 or 0, maybe it would be best encapsulation wise to turn that into a true or false. Your call though.